app/admin/RTMediaUploadTermsAdmin.php

```diff
@@ -0,0 +1,209 @@
+		 * Configure admin options to render
+		 *
+		 * @param  array $render_options  Rendering according to selected options.
+		 * @param  array $options         Options selected in settings.
+		 * @return array $render_option
+		 */
+		public function admin_setting_add_terms_option( $render_options, $options ) {
+			$render_options[ 'general_enable_upload_terms' ]    = array(
+				'title'    => __( 'Show "Terms of Service" checkbox on upload screen', 'buddypress-media' ),
+				'callback' => array( 'RTMediaFormHandler', 'checkbox' ),
+				'args'     => array(
+					'key'   => 'general_enable_upload_terms',
+					'value' => $options[ 'general_enable_upload_terms' ],
+					'desc'  => __( 'User have to check the terms and conditions before uploading the media.', 'buddypress-media' ),
+				),
+				'group'    => 40,
+			);
+			$render_options['activity_enable_upload_terms']   = array(
+				'title'    => __( 'Show "Terms of Service" checkbox on activity screen', 'buddypress-media' ),
+				'callback' => array( 'RTMediaFormHandler', 'checkbox' ),
+				'args'     => array(
+					'key'   => 'activity_enable_upload_terms',
+					'value' => $options[ 'activity_enable_upload_terms' ],
+					'desc'  => __( 'User have to check the terms and conditions before uploading the media.', 'buddypress-media' ),
+				),
+				'group'    => 40,
+			);
+			$render_options['general_upload_terms_page_link'] = array(
+				'title'    => __( 'Link for "Terms of Service" page', 'buddypress-media' ),
+				'callback' => array( 'RTMediaFormHandler', 'textbox' ),
+				'args'     => array(
+					'key'   => 'general_upload_terms_page_link',
+					'value' => $options[ 'general_upload_terms_page_link' ],
+					'desc'  => __( 'Link to the terms and condition page where user can read terms and conditions.', 'buddypress-media' ),
+				),
+				'group'    => 40,
+			);
+			// add extra field for admin setting.
+			$render_options[ 'general_upload_terms_message' ]              = array(
+				'title'    => __( 'Terms of Service Message', 'buddypress-media' ),
+				'callback' => array( 'RTMediaFormHandler', 'textbox' ),
+				'args'     => array(
+					'key'   => 'general_upload_terms_message',
+					'value' => isset( $options[ 'general_upload_terms_message' ] ) ? $options[ 'general_upload_terms_message' ] : $this->upload_terms_message,
+					'desc'  => __( 'Add Terms of Service Message.', 'buddypress-media' ),
+				),
+				'group'    => 40,
+			);
+			$render_options['general_upload_terms_error_message']        = array(
+				'title'    => __( 'Error Message', 'buddypress-media' ),
+				'callback' => array( 'RTMediaFormHandler', 'textbox' ),
+				'args'     => array(
+					'key'   => 'general_upload_terms_error_message',
+					'value' => isset( $options[ 'general_upload_terms_error_message' ] ) ? $options[ 'general_upload_terms_error_message' ] : $this->upload_terms_error_message,
+					'desc'  => __( 'Display Error Message When User Upload Media Without selecting checkbox .', 'buddypress-media' ),
```

Comment:
Maintain the consistency with the capitalization of characters in this sentence.